stages/01_adrecs.py

```diff
@@ -27,12 +51,18 @@ def retrieve_data(self):
           'drug_adr_relations_with_quantitative_features.txt.gz': 'http://bioinf.xmu.edu.cn/ADReCS/download/v3.3/ADReCS_Drug_ADR_relations_quantification_v3.3.txt.gz',
         }
 
-        _ = [ urllib.request.urlretrieve(link, os.path.join('data', data)) for data, link in data_links.items() ]
+        _ = [ urllib.request.urlretrieve(link, os.path.join('cache', data)) for data, link in data_links.items() ]
 
     def process_data(self):
 
-        _, _, files = next(os.walk('data'))
-        files = [ os.path.join('data', file) for file in files ]
+        '''
+
+        Process the data stored in the ADRECS, takes in gzip and xlsx and converts to parquet
+
+        '''
+
+        _, _, files = next(os.walk('cache'))
+        files = [ os.path.join('brick', file) for file in files ]
 
         for file in files:
```

Comment:
You should be able to do:
```python
df = pd.read_csv('download/v3.3/Drug_ADR_v3.3.txt.gz', sep='\t')
```
No need for decompression and manually extracting the headers.

Reply:
There was some encode if you mean why I did the decode('utf-8') and popped the headers? or iterating through the directory?

Reply:
`read_csv()` can also take an `encoding` parameter: <https://pandas.pydata.org/pandas-docs/version/2.2/reference/api/pandas.read_csv.html>.